tests/MariaDbEventStoreTestCase.php

```diff
@@ -62,7 +62,7 @@ protected function setUp(): void
 
     /**
      * @test
-     * @medium
+     * @large
```

Comment:
Gives this test more time to succeed. @medium gives 10 seconds, but is sometimes too short